src/frontend/src/test-e2e/selenium.test.ts

```diff
@@ -41,6 +41,15 @@ const DEMO_APP_URL = "http://localhost:8080/";
 const DEVICE_NAME1 = "Virtual WebAuthn device";
 const DEVICE_NAME2 = "Other WebAuthn device";
 
+const screenshots = new Screenshots("screenshots");
```

Comment:
We could construct this inside the screenshot test, because the numbering is tied to that test. Maybe in the future there will be an additional test that also takes screenshots in a different folder with its own ordering

Reply:
Ah yes, makes sense!